slave/tests/hostnamectl/Makefile

```diff
@@ -0,0 +1,65 @@
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
In the long term, I hope we can have just one canonical location so we don't need to "sync" things.